core/services/fluxmonitorv2/flux_monitor.go

```diff
@@ -490,19 +501,15 @@ func (fm *FluxMonitor) processLogs() {
 			if !isFlagLowered {
 				fm.pollManager.Hibernate()
 			}
-			err = broadcast.MarkConsumed()
```

Comment:
Don't we still need to mark all of these as consumed?

Reply:
you're right, need to anyways in the case of all the early returns

Reply:
hmm you're right we need to mark them consumed in the early return cases as well, but we don't want to do it unconditionally - if the insert or ethtx fails, we'd want to reprocess

Reply:
Sorry you mean the flag lowered case? Yeah it gets marked consumed inside pollIfEligible, we don't want to unconditionally mark it consumed incase it was supposed to poll but didnt because the tx didnt go through